internal/addon/manifests/charts/mcoa/templates/namespace.yaml

```
@@ -0,0 +1,11 @@
+{{- if .Values.enabled }}
```

For COO resources we should have this

```suggestion
{{- if .Values.installCOO }}
```

Then on the `internal/addon/manifests/charts/mcoa/templates/_helpers.tpl` we should have the following, as new projects want to use COO they should add themselves to the `if`
```
{{- define "mcoahelm.installCOO" -}}
{{- if and .Values.enabled .Values.analytics.incidentDetection.enabled -}}
true
{{- else -}}
false
{{- end -}}
{{- end -}}
```